bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -0,0 +1,454 @@
+      url='img/firefox/challenge-the-default/logo-ctd.svg',
+        optional_attributes={
+          'loading': 'eager',
+          'height': '24',
+          'class': 'ctd-sub-nav-image mobile',
+          'alt': 'Challenge the default'
+      })
+    }}
+  </div>
+  <section class="ctd-hero-wrapper">
+    <div class="hidden hero-easter-egg">
+      {{ resp_img(
+        url="img/firefox/challenge-the-default/hero-easter-egg.svg",
+        optional_attributes={
+          'alt': "",
+          'loading': "lazy",
+          'height': '196',
+          'width': '196'
+        })
+      }}
+        <p>Hier gibt es nichts zu sehen. Also … wirklich gar nichts</p>
+    </div>
+    <div class="c-ctd-hero mzp-l-content mzp-t-content-xl">
+      <svg class="svg" width="0" height="0">
+        <clipPath id="hero-clip-path-mobile" clipPathUnits="objectBoundingBox">
+          <path d="M0.028,0 H0.992 L1,1 L0,0.973 V0.329 L0.028,0"></path>
+        </clipPath>
+        <clipPath id="hero-clip-path" clipPathUnits="objectBoundingBox">
+          <path d="M0.011,0.985 L0,0 L1,0.02 V0.663 L0.968,1 L0.011,0.985"></path>
+        </clipPath>
+        <clipPath id="features-clip-path" clipPathUnits="objectBoundingBox">
+          <path d="M0.763,0 H0 V0.453 L0.037,0.958 L0.444,1 L1,0.965 V0.819 L0.971,0.352 L0.984,0.033 L0.763,0"></path>
+        </clipPath>
+        <clipPath id="kitten-clip-path" clipPathUnits="objectBoundingBox">
+          <path d="M0.326,0.056 H0.032 L0,0.306 L0.032,0.944 L1,1 V0.794 L0.937,0.056 L0.326,0 V0.056"></path>
+        </clipPath>
+      </svg>
+      <div class="hero-wrapper">
+        <div class="c-hero-top">
+          <div class="c-hero-top-images">
+              {{ resp_img(
+                url='img/firefox/challenge-the-default/logo-firefox.svg',
+                optional_attributes={
+                  'loading': 'eager',
+                  'alt': 'Firefox',
+                })
+              }}
+              {{ resp_img(
+                url='img/firefox/challenge-the-default/logo-ctd.svg',
+                optional_attributes={
+                  'loading': 'eager',
+                  'alt': 'Challenge the default'
+                })
+              }}
+          </div>
+          <div class="c-hero-top-controls">
+              <button class="hero-control-btn minimize"></button>
+              <button class="hero-control-btn expand"></button>
+              <button class="hero-control-btn close"></button>
+          </div>
+        </div>
+        <div class="hero-content-wrapper">
+          <div class="c-hero-text">
+            <h1>Der Browser, den du selber wählst</h1>
+            <p>Wenn du Firefox wählst, triffst du eine bewusste Entscheidung, anstatt einfach das Erstbeste zu nehmen. Besserer
+          Datenschutz, mehr Transparenz, und ein gesünderes Web – deine Entscheidung.</p>
+            <button class="is-not-default mzp-c-button mzp-t-lg mzp-js-notification-trigger">Jetzt Firefox wählen</button>
```

Comment:
Also: just thinking out loud, but might it be simpler to link to `set-as-default/thanks` here, rather than duplicate all of the logic again here on this page?

Reply:
omg, i didn't even consider to let that page handle all of the set-as-default logic. 🤞🏻 I'll give it a go!